node/peer.js

```diff
@@ -62,7 +63,13 @@ function TChannelPeer(channel, hostPort, options) {
         );
     }
 
-    self.setScoreStrategy(PreferOutgoing);
+    self.preferOutgoing = self.options.preferOutgoing;
```

Comment:
let's default to `preferOutgoing` and manually mark the ringpop sub channel as preferNone.